apps/editing-toolkit/editing-toolkit-plugin/full-site-editing-plugin.php

```diff
@@ -357,4 +342,40 @@ function load_wpcom_global_styles() {
 
 	require_once __DIR__ . '/wpcom-global-styles/index.php';
 }
-add_action( 'plugins_loaded', __NAMESPACE__ . '\load_wpcom_global_styles' );
+
+/**
+ * Load features of the ETK plugin.
+ */
+function load_editing_toolkit_features() {
+	/**
+	 * Turn off the ETK plugin features if the Jetpack_Mu_Wpcom is available and its version is v5.54.0-alpha and above.
+	 */
+	if ( class_exists( '\Automattic\Jetpack\Jetpack_Mu_Wpcom', false ) && version_compare( \Automattic\Jetpack\Jetpack_Mu_Wpcom::PACKAGE_VERSION, '5.54.0-alpha', '>=' ) ) {
```

Comment:
Would it be safe to check the next one or two versions like 5.55.0 or 5.56.0? Any thoughts?

Reply:
This seems fine to me, any reason why would it be safer? Is it because they'd be minor versions instead of alpha?

Reply:
I'm afraid of the revert 🙈

Reply:
Would there be any issues with bumping the version compare to 5.55, which might cause ETK and jetpack-mu-plugin coexisting? I think that should be fine, no? We have flags in place to prevent this, so this PR should be more of a failsafe, CMIIW.

Reply:
No. It would be fine to use any versions before we remove the flags in jetpack-mu-wpcom